lib/routes/account.js

```diff
@@ -741,6 +815,18 @@ module.exports = function (
           }
           return P.resolve(response)
         }
+
+        function gateSigninUnblock (err) {
+          // customs.check will always add these properties if the
+          // customs server has not rate-limited unblock. Nonetheless,
+          // we shouldn't signal to the content-server that it is
+          // possible to unblock the user if the feature is not allowed.
+          if (!allowSigninUnblock && err.output && err.output.payload) {
```

Comment:
Do we need an activity event if the request is blocked, or does that happen for free?